modules/ivc_champva/app/models/ivc_champva/vha_10_10d.rb

```diff
@@ -38,9 +38,36 @@ def metadata
       }
     end
 
+    # rubocop:disable Layout/LineLength, Style/IfUnlessModifier
```

Comment:
Why are these disables being added here?

Reply:
I wanted to disable linelength and the unless modifier was better than using  || or &&

Reply:
My rubocop on my IDE is not catching these errors but on the CICD it is.